connectors/sources/mssql.py

```diff
@@ -255,6 +255,9 @@ async def ping(self):
     async def get_tables_to_fetch(self, is_filtering=False):
         tables = configured_tables(self.tables)
         if is_wildcard(tables) or is_filtering:
+            self._logger.info(
+                "Fetching all tables as the configuration field 'tables' is set to '*' or advanced sync rule is enabled."
+            )
```

Comment:
```suggestion
            self._logger.info(
                "Fetching all tables as the configuration field 'tables' is set to '*' or advanced sync rules are enabled."
            )
```

Reply:
I think we should also split the log message without concatenating two possible reasons with `or`. This adds ambiguity IMO.